include/dt-bindings/clock/imxrt1170-clock.h

```diff
@@ -47,20 +47,22 @@
 #define IMXRT1170_CLK_ROOT_LPUART1		33
 #define IMXRT1170_CLK_ROOT_ENET1		34
 #define IMXRT1170_CLK_ROOT_USDHC1		35
+#define IMXRT1170_CLK_ROOT_ADC1 		36
+#define IMXRT1170_CLK_ROOT_ADC2 		37
 
 /* periph gates */
-#define IMXRT1170_CLK_M7			36
-#define IMXRT1170_CLK_SEMC			37
-#define IMXRT1170_CLK_GPT1			38
-#define IMXRT1170_CLK_CAN1			39
-#define IMXRT1170_CLK_CAN2			40
-#define IMXRT1170_CLK_CAN3			41
-#define IMXRT1170_CLK_LPUART1			42
-#define IMXRT1170_CLK_ENET1			43
-#define IMXRT1170_CLK_USDHC1			44
-#define IMXRT1170_CLK_BUS			45
-#define IMXRT1170_CLK_USB			46
+#define IMXRT1170_CLK_M7			38
+#define IMXRT1170_CLK_SEMC			39
+#define IMXRT1170_CLK_GPT1			40
+#define IMXRT1170_CLK_CAN1			41
+#define IMXRT1170_CLK_CAN2			42
+#define IMXRT1170_CLK_CAN3			43
+#define IMXRT1170_CLK_LPUART1			44
+#define IMXRT1170_CLK_ENET1			45
+#define IMXRT1170_CLK_USDHC1			46
+#define IMXRT1170_CLK_BUS			47
+#define IMXRT1170_CLK_USB			48
```

Comment:
Shouldn't we add IDs for the ADC gates (LPCG55 and LPCG56 as per the processor reference manual) ?